src/mockserver.js

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs';
+
+import { logger } from '@user-office-software/duo-logger';
+import { mockServerClient } from 'mockserver-client';
+
+async function mockserver() {
+    var callback = function (request) {
+        if (request.method === 'POST') {
+            const name = String(request.body.xml);
+            let regexp = '<tns:(.*?)>';
+            const test = name.match(regexp);
+            const method = test[1];
+
+            if (
+                name.includes('getBasicPersonDetailsFromUserNumber') ||
+                name.includes('getSearchableBasicPeopleDetailsFromUserNumbers')
+            ) {
+                let calls = '';
+                name.includes('getSearchableBasicPeopleDetailsFromUserNumbers')
+                    ? (calls = '<UserNumbers>(.*?)<')
+                    : (calls = '<UserNumber>(.*?)<');
+                const match = name.match(calls);
+
+                const file = JSON.parse(
+                    fs.readFileSync(
+                        'src/responses/user/' + method + '/' + match[1] + '.txt',
+                        'utf8'
+                    )
+                );
+                const request1 = file.body.xml;
+
+                return {
+                    body: request1,
+                };
+            }
+            if (name.includes('getSearchableBasicPersonDetailsFromEmail')) {
+                let regexp = '<Email>(.*?)<';
+                const match = name.match(regexp);
+
+                const file = JSON.parse(
+                    fs.readFileSync(
+                        'src/responses/user/' + method + '/' + match[1] + '.txt',
+                        'utf8'
+                    )
+                );
+                const request1 = file.body.xml;
+
+                return {
+                    body: request1,
+                };
+            }
+            if (
+                name.includes('getBasicPeopleDetailsFromUserNumbers') &&
+                name.includes('<UserNumbers>')
+            ) {
+                const file = JSON.parse(
+                    fs.readFileSync('src/responses/user/notEmptyResponse' + '.txt', 'utf8')
+                );
+                const request1 = file.body.xml;
+
+                return {
+                    body: request1,
+                };
+            }
+            const file = JSON.parse(
+                fs.readFileSync('src/responses/user/' + method + '.txt', 'utf8')
+            );
+            const request1 = file.body.xml;
+
+            return {
+                body: request1,
+            };
+        }
+    };
+    mockServerClient('mockServer', 1080)
+        .mockWithCallback(
+            {
+                method: 'POST',
+                path: '/ws/UserOfficeWebService',
+            },
+            callback,
+            {
+                unlimited: true,
+            }
+        )
+        .then(
+            function () {
+                logger.logInfo('expectation created, callabck', {});
```

Comment:
We should have a more specific success message, like "Created callback for POST requests". The success message for the `GET` expectation later on should also be something like this